.chronus/changes/python-moveToVendorFolder-2025-3-17-13-43-15.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@typespec/http-client-python"
+---
+
+Reorder generated `_vendor` file into a `_utils_` folder
```

Comment:
This PR influences all SDKs including DPG SDK and MPG SDK. Please make an related PR to show the code diff in autorest.python repo.

Reply:
yep, have been working on that PR too, won't merge this until that PR is green https://github.com/Azure/autorest.python/pull/3077